pitop/processing/algorithms/ball_detect.py

```diff
@@ -126,30 +148,28 @@ def __print_fps(self):
         print(f"[INFO] Approx. FPS: {self._fps.fps():.2f}")
 
     def __draw_ball_position(self, frame, ball):
-        self.cv2.circle(frame, ball.center, ball.radius, (0, 255, 255), 2)
-        self.cv2.circle(frame, ball.center, 5, tuple_for_color_by_name(ball.color, bgr=True), -1)
+        self.cv2.circle(frame, ball.center_points_cv[0], ball.radius, (0, 255, 255), 2)
```

Comment:
Why did we drop `.center`?

Reply:
Because it's in the wrong coordinates for OpenCV at this point in the code